econml/_ortho_learner.py

```diff
@@ -0,0 +1,250 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Double ML.
+
+"Double Machine Learning" is an algorithm that applies arbitrary machine learning methods
+to fit the treatment and response, then uses a linear model to predict the response residuals
+from the treatment residuals.
+
+"""
+
+import numpy as np
+import copy
+from warnings import warn
+from .utilities import (shape, reshape, ndim, hstack, cross_product, transpose,
+                        broadcast_unit_treatments, reshape_treatmentwise_effects,
+                        StatsModelsLinearRegression, LassoCVWrapper)
+from sklearn.model_selection import KFold, StratifiedKFold, check_cv
+from sklearn.linear_model import LinearRegression, LassoCV
+from sklearn.preprocessing import (PolynomialFeatures, LabelEncoder, OneHotEncoder,
+                                   FunctionTransformer)
+from sklearn.base import clone, TransformerMixin
+from sklearn.pipeline import Pipeline
+from sklearn.utils import check_random_state
+from .cate_estimator import (BaseCateEstimator, LinearCateEstimator,
+                             TreatmentExpansionMixin, StatsModelsCateEstimatorMixin)
+from .inference import StatsModelsInference
+
+
+def _crossfit(model, folds, *args, **kwargs):
+    model_list = []
+    for idx, (train_idxs, test_idxs) in enumerate(folds):
+        model_list.append(clone(model, safe=False))
+
+        args_train = ()
+        args_test = ()
+        for var in args:
+            args_train += (var[train_idxs],)
```

Comment:
Shouldn't this also have an `if var is not None:` guard, as with the keyword arguments?